tmail-backend/blob/blobid-list/src/test/java/com/linagora/tmail/blob/blobid/list/BlobStoreModulesChooserTest.java

```diff
@@ -0,0 +1,35 @@
+package com.linagora.tmail.blob.blobid.list;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import org.apache.james.blob.aes.CryptoConfig;
+import org.junit.jupiter.api.Test;
+
+public class BlobStoreModulesChooserTest {
+
+    @Test
+    void provideBlobStoreShouldReturnNoEncryptionWhenNoneConfigured() {
+        assertThat(BlobStoreModulesChooser.chooseModules(BlobStoreConfiguration.builder()
+            .disableCache()
+            .deduplication()
+            .noCryptoConfig()
+            .disableSingleSave()))
+            .filteredOn(module -> module instanceof BlobStoreModulesChooser.NoEncryptionModule)
+            .hasSize(1);
+    }
+
+    @Test
+    void provideBlobStoreShouldReturnEncryptionWhenConfigured() {
+        assertThat(BlobStoreModulesChooser.chooseModules(BlobStoreConfiguration.builder()
+            .disableCache()
+            .passthrough()
+            .cryptoConfig(CryptoConfig.builder()
+                .password("myPass".toCharArray())
+                // Hex.encode("salty".getBytes(StandardCharsets.UTF_8))
```

Comment:
Why?

Reply:
I just copied it from old code `org.apache.james.modules.blobstore.BlobStoreModulesChooserTest`